ladybug/skymodel.py

```diff
@@ -216,3 +291,880 @@ def sky_temperature(horiz_ir, dry_bulb):
     return sky_temp
+
+
+"""DIRECT AND DIFFUSE SPLITTING FROM GLOBAL HORIZONTAL"""
+"""The following code is a modified version of the PVLib python library.
+
+PVLib
+Copyright (c) 2013-2018, Sandia National Laboratories and pvlib python Development Team
+All rights reserved.
+
+Developed at Sandia National Laboratories, PVLib implements
+many of the models and methods developed at the Lab.
+More information can be found on the pvlib-python github:
+https://github.com/pvlib/pvlib-python
+"""
+
+
+def dirint(ghi, altitudes, doys, pressures, use_delta_kt_prime=True,
+           temp_dew=None, min_sin_altitude=0.065, min_altitude=3):
+    """
+    Determine DNI from GHI using the DIRINT modification of the DISC
+    model.
+
+    Implements the modified DISC model known as "DIRINT" introduced in
+    [1]. DIRINT predicts direct normal irradiance (DNI) from measured
+    global horizontal irradiance (GHI). DIRINT improves upon the DISC
+    model by using time-series GHI data and dew point temperature
+    information. The effectiveness of the DIRINT model improves with
+    each piece of information provided.
+
+    The pvlib implementation limits the clearness index to 1.
+
+    Args:
+        ghi : array-like
+            Global horizontal irradiance in W/m^2.
+        altitudes : array-like
+            True (not refraction-corrected) solar altitude angles in decimal
+            degrees.
+        doys : array-like
+            Integers representing the day of the year.
+        pressures : array-like
+            The site pressure in Pascal. Pressure may be measured or an
+            average pressure may be calculated from site altitude.
+        use_delta_kt_prime : bool, default True
+            If True, indicates that the stability index delta_kt_prime is
+            included in the model. The stability index adjusts the estimated
+            DNI in response to dynamics in the time series of GHI. It is
+            recommended that delta_kt_prime is not used if the time between
+            GHI points is 1.5 hours or greater. If use_delta_kt_prime=True,
+            input data must be Series.
+        temp_dew : None or array-like, default None
+            Surface dew point temperatures, in degrees C. Values of temp_dew
+            must be numeric. If temp_dew is not provided, then dew point
+            improvements are not applied.
+        min_sin_altitude : numeric, default 0.065
+            Minimum value of sin(altitude) to allow when calculating global
+            clearness index `kt`. Equivalent to altitude = 3.727 degrees.
+        min_altitude : numeric, default 87
+            Minimum value of altitude to allow in DNI calculation. DNI will be
+            set to 0 for times with altitude values smaller than `min_altitude`.
+
+    Returns:
+        dni : array-like
+            The modeled direct normal irradiance in W/m^2 provided by the
+            DIRINT model.
+
+    Notes
```

Comment:
I didn't want to comment of every single Notes and References. As a generic rule they should not be separated down here. Otherwise they will be missed from the documentation.

Reply:
Ok. This docstring was just copy / pasted from [PVLib](https://github.com/pvlib/pvlib-python), as I mentioned earlier.  Let me know the right way to add them to "expanded description" and I will implement it.

Reply:
I answered to a similar question above. It's mainly because this structure won't be parsed by documentation libraries correctly.